src/index.js

```diff
@@ -53,9 +53,40 @@ function isReactClass(superClass, scope, globalOptions) {
   return answer
 }
 
+function areSetsEqual(set1, set2) {
+  if (set1.size !== set2.size) {
+    return false
+  }
+
+  return !Array.from(set1).some(item => !set2.has(item))
+}
+
 export default function(api) {
   const { template, types, traverse } = api
 
+  const nestedIdentifiers = new Set()
+  const removedPaths = new Set()
+  const collectNestedIdentifiers = {
+    Identifier(path) {
+      if (path.parent.type === 'MemberExpression') {
+        return
+      }
+
+      if (
+        path.parent.type === 'ObjectProperty' &&
+        (path.parent.key === path.node || path.parent.shorthand)
+      ) {
+        return
+      }
+
+      if (path.parent.type === 'CallExpression' && path.parent.callee === path.node) {
```

Comment:
I think we don't actually need this check. I don't actually remember why I added it in the first place.